package.json

```diff
@@ -1,37 +1,36 @@
 {
   "name": "linterhub-atom",
-  "main": "./lib/linterhub-atom",
-  "version": "0.2.3",
+  "main": "./lib/main",
+  "version": "0.3.1",
   "description": "Linterhub extension for Atom",
   "keywords": [],
   "repository": "https://github.com/binore/linterhub-atom",
   "license": "MIT",
   "engines": {
-    "atom": ">=1.0.0 <2.0.0"
+    "atom": ">=1.4.0 <2.0.0"
   },
   "dependencies": {
     "fs": "0.0.1-security",
     "path": "^0.12.7",
-    "linterhub-ide": "^0.3.7",
+    "linterhub-ide": "^0.4.3",
     "dom-listener": "^0.1.2",
     "atom-linter": "^9.0.0",
-    "atom-package-deps": "^4.4.1"
+    "atom-package-deps": "^4.4.1",
```

Comment:
If you bump the minimum version of this to `^4.5.0` you can change your Linter package dependency to `"linter:2.0.0"` and ensure that your users are on the required version of Linter.